testRange.c

```diff
@@ -0,0 +1,21 @@
+#include <stdio.h>
+#include <assert.h>
+#include "detectRange.h"
+
+void TEST_CASE_01()
+{
+    int data[2]={4,5};
```

Comment:
The test cases need to handle single element array, dual element array with same repeating numbers.
Also test cases to verify the content of the range need to be included.
Since you already have the data structure verification is easy 👍

Reply:
@prithamvshetty  is it necessary to handle multi dimensional array? in the assignment stating like "array of variables" this means one dimensional array right ?...i will be extending the test

Reply:
input is series of values i.e. single dimensional array of values. Hence handling of multidimensional array is not needed.